lib/src/ui/watch/watch_screen.dart

```diff
@@ -0,0 +1,200 @@
+        actions: [
+          IconButton(
+            icon: isSoundMuted
+                ? const Icon(Icons.volume_off)
+                : const Icon(Icons.volume_up),
+            onPressed: () => ref.read(muteSoundPrefProvider.notifier).toggle(),
+          ),
+        ],
+      ),
+      body: RefreshIndicator(
+        key: _andoirdRefreshKey,
+        onRefresh: () => ref.refresh(streamerListProvider.future),
+        child: _WatchScaffold(
+          child: ListView(
+            padding: Styles.verticalBodyPadding,
+            children: [const _WatchTvWidget(), StreamerWidget()],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildIos(BuildContext context, WidgetRef ref) {
+    final isSoundMuted = ref.watch(muteSoundPrefProvider);
+    return CupertinoPageScaffold(
+      child: _WatchScaffold(
+        child: CustomScrollView(
+          slivers: [
+            CupertinoSliverNavigationBar(
+              trailing: CupertinoButton(
+                padding: EdgeInsets.zero,
+                child: isSoundMuted
+                    ? const Icon(CupertinoIcons.volume_off)
+                    : const Icon(CupertinoIcons.volume_up),
+                onPressed: () =>
+                    ref.read(muteSoundPrefProvider.notifier).toggle(),
+              ),
+            ),
+            CupertinoSliverRefreshControl(
+              onRefresh: () => ref.refresh(streamerListProvider.future),
+            ),
+            SliverSafeArea(
+              top: false,
+              sliver: SliverPadding(
+                padding: Styles.verticalBodyPadding,
+                sliver: SliverList(
+                  delegate: SliverChildListDelegate(
+                    [const _WatchTvWidget(), StreamerWidget()],
+                  ),
+                ),
+              ),
+            )
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _WatchTvWidget extends ConsumerWidget {
+  const _WatchTvWidget();
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final pieceSet = ref.watch(pieceSetPrefProvider);
+    final currentTab = ref.watch(currentBottomTabProvider);
+    final tvStream = currentTab == BottomTab.watch
+        ? ref.watch(tvStreamProvider)
+        : const AsyncLoading<FeaturedPosition>();
+    final featuredGame = ref.watch(featuredGameProvider);
+    return ListSection(
+      hasLeading: true,
+      header: const Text('Watch Top Games'),
+      onHeaderTap: () => Navigator.of(context).push<void>(
+        MaterialPageRoute(builder: (context) => const TvScreen()),
+      ),
+      children: [
+        tvStream.when(
+          data: (position) {
+            final boardData = BoardData(
+              interactableSide: InteractableSide.none,
+              orientation: featuredGame?.orientation.cg ?? Side.white,
+              fen: position.fen,
+              lastMove: position.lastMove?.cg,
+            );
+            return Center(
+              child: ConstrainedBox(
```

Comment:
Is it not the same as `SizedBox.square(350)`?

Perhaps we should also adapt to different screen sizes (including tablets), but that can be done later I guess.